### ISSUE TITLE
Unable to convert a structured question summarized by date to SQL

### ISSUE THREAD

**Issue by alxnddr** [metabase/metabase#21591]
**Describe the bug**
Unable to convert a structured question summarized by date to SQL

**Logs**
```
Uncaught TypeError: query.breakouts is not a function
    at TimeseriesFilterWidget.UNSAFE_componentWillReceiveProps (TimeseriesFilterWidget.jsx:32:1)
    at callComponentWillReceiveProps (react-dom.development.js:12976:1)
    at updateClassInstance (react-dom.development.js:13178:1)
    at updateClassComponent (react-dom.development.js:17107:1)
```

**To Reproduce**
1. Go to New -> Question -> Sample Database -> Orders
2. Click Visualize -> Summarize -> Created At by month
3. Click show editor -> View in SQL -> Convert this question to SQL
4. The page is blank now

**Expected behavior**
Converts the question to native form using the sql query from the modal

**Information about your Metabase Installation:**

Latest `master`

**Comment by flamber** [comment]
@alxnddr I cannot reproduce. Could this be related to events, which was just fixed? #21611

**Comment by alxnddr** [comment]
@flamber seems like it is fixed by https://github.com/metabase/metabase/pull/21589/files#diff-9265ff5fd4e44287bf7c9a998c90f0c8c26478760a5f3549901e4637d7f23db4L32
